rdfunit-junit/src/main/java/org/aksw/rdfunit/junit/RdfUnitJunitRunner.java

```diff
@@ -220,12 +220,17 @@ protected Description describeChild(RdfUnitJunitTestCase child) {
                 child.getTestInputMethod().getDeclaringClass(),
                 String.format(
                         "[%s] %s",
-                        child.getTestInputMethod().getMethod().getName(),
+                        getTestInputBaseName(child.getTestInputMethod()),
                         child.getTestCase().getAbrTestURI()
                 )
         );
     }
 
+    private String getTestInputBaseName(FrameworkMethod method) {
```

Comment:
I thought of it as just the base name - the other part of the name is basically the RDFUnit URI. So the base name will be the same for a number of tests.